### DISCUSSION TITLE
What message we want to send with the V3 Release?

### DISCUSSION THREAD

**Discussion by balhar-jakub** [zowe/community#2291] [General]
The Zowe V3 is coming and with the V3 we should prepare the material for Marketing and sharing the V3. This discussion is for us to figure out what we want to prepare.

**Comment by RASakach** [comment]
Notes & some considerations from todays (8/27) ZAC discussion:

Announcement Channels:
- OMP
- Social Media (LinkedIn, X, etc.)
- OMP Distribution Lists (ZOWE USER, etc.)
- Zowe.org landing page

Types of content:
- Press Release, Blog, and  ? TBD
- Simple announcement letter
- Simple announcement banner

Key communication topics, highlights, and overall messaging: 
- What is the headline for Zowe v3? 
   -  The addition of the NEW CORE components (new products brought into the Zowe fold..)
   - ….  the Story continues, still Open, Simple, and Familiar AND mature 
   - YoY increase in growth, supported by Arcati statistics (note: limit to 100 customers but viable sample size) 
   - Maturity = stability, with V3, V2 becomes the stable version, will be around for a long time (Zowe is now Mature & stable;  evident by far less influential breaking changes in a major version) 
   - Community growth & expansion continues (i.e. IntelliJ Squad) - 
   - Security:  strengthening our stance on our packaging and our overall Security stance - how are we above?  How far we are going compared to typical analysis

**Comment by RASakach** [comment]
From the OMP Marketing Meeting:
- A new article:  Enterprise Journal type
- OMP Blog (Joe leading with help from Jakub & Rose)
- Quotes from Leaders at various participating companies:  Broadcom, IBM, IBA, Rocket
- TechStrong TV event planned w/Jakub
- TFIR video with at least 1 of the Squad leads
-
